packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -47,12 +47,26 @@ abstract class ConstrainedLayoutBuilder<ConstraintType extends Constraints> exte
   /// The builder must not return null.
   final Widget Function(BuildContext context, ConstraintType constraints) builder;
 
+  /// Whether [builder] needs to be called again even if the layout constraints
+  /// are the same.
+  ///
+  /// When this widget is rebuilt, it is most likely that we need to call
```

Comment:
nit: avoid "we" -> https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple

Reply:
Done